lib/whatsapp_sdk/api/responses/business_profile_data_response.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+# typed: strict
+
+require_relative "data_response"
+
+module WhatsappSdk
+  module Api
+    module Responses
+      class BusinessProfileDataResponse < DataResponse
+        sig { returns(String) }
+        attr_accessor :about
+
+        sig { returns(String) }
+        attr_accessor :address
+
+        sig { returns(String) }
+        attr_accessor :description
+
+        sig { returns(String) }
+        attr_accessor :email
+
+        sig { returns(String) }
+        attr_accessor :messaging_product
+
+        sig { returns(String) }
+        attr_accessor :profile_picture_url
```

Comment:
Could you add `websites` too please?

Reply:
Yes sure.

Reply:
Have added it in #56 as per the discussion on this PR.